lib/ansible/modules/database/postgresql/postgresql_db.py

```diff
@@ -209,13 +225,24 @@ def db_create(cursor, db, owner, template, encoding, lc_collate, lc_ctype):
                 'Changing LC_CTYPE is not supported.'
                 'Current LC_CTYPE: %s' % db_info['lc_ctype']
             )
-        elif owner and owner != db_info['owner']:
-            return set_owner(cursor, db, owner)
         else:
-            return False
+            if owner and owner != db_info['owner']:
```

Comment:
This whole block is a little strange. It might be clearer to write it as:

```
changed = False

if owner and owner != db_info['owner']:
    changed = set_owner(cursor, db, owner)

if conn_limit and conn_limit != str(…):
    changed = set_conn_limit(cursor, db, conn_limit)

return changed
```